astronomer/providers/microsoft/azure/hooks/data_factory.py

```diff
@@ -1,9 +1,10 @@
+"""This module contains the Azure Data Factory hook's asynchronous implementation."""
 import inspect
 from functools import wraps
 from typing import Any, Optional, TypeVar, Union, cast
 
 from airflow.exceptions import AirflowException
-from airflow.providers.microsoft.azure.hooks.data_factory import AzureDataFactoryHook
+from airflow.providers.microsoft.azure.hooks.data_factory import AzureDataFactoryHook, get_field
```

Comment:
Importing this will fail on v4 of the provider I think - is that a problem? Is this meant to only work with v5+?

Reply:
It's meant to support both versions. Thanks, @ashb , I will create another PR to fix this.

Reply:
Created a PR https://github.com/astronomer/astronomer-providers/pull/900 to fix this.